h2o-core/src/main/java/water/rapids/ast/prims/mungers/AstPivot.java

```diff
@@ -0,0 +1,79 @@
+package water.rapids.ast.prims.mungers;
+
+import water.*;
+import water.fvec.*;
+import water.rapids.Rapids;
+import water.rapids.Env;
+import water.rapids.vals.ValFrame;
+import water.rapids.ast.AstPrimitive;
+import water.rapids.ast.AstRoot;
+import water.util.VecUtils;
+
+public class AstPivot extends AstPrimitive {
+  @Override
+  public String[] args() {
+    return new String[]{"ary", "index", "column", "value"}; //the array and name of columns
+  }
```

Comment:
In particular, a pivot table usually requires 1 more parameter: `aggregation` -- to specify what kind of aggregation function should be applied to the values. I'm not sure what kind of aggregation you're applying right now -- is it `first`?